### ISSUE TITLE
Dockerfile and documentation for clean build environment

### ISSUE THREAD

**Issue by 1-21gigasats** [bitcoin-tools/diyjade#15]
When flashing on my desktop computer i prefer to have an isolated build environment. This is why i created a Dockerfile that builds a container that includes all needed dependencies and executes the flash script. The tty is made available with the device argument. Do you think this useful for others? Should i create a pr with a documentation how to use it?

**Comment by bitcoin-tools** [comment]
Yes, this would be helpful. Can you please create a Dockerfile MR like you're thinking? How would you handle the five different scripts?
